causal_testing/json_front/README.md

```diff
@@ -0,0 +1,32 @@
+# JSON Causal Testing Framework Frontend
```

Comment:
This file is going to be a bit tricky to find here in the package. It might be better merged with the main readme for now.

Reply:
Would it be a good idea to move this readme to the readthedocs and point to it from the main readme?

Reply:
That would also work. Examples / tutorials should be prominent.